### PR TITLE
fixing-typo

### PR DESCRIPTION
**Pull request by sonaljain067**
**Description**

This PR fixes some grammatical and typos.

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [✔️] Yes, I signed my commits.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ For further information, refer to the [Meshery Docs](https://docs.meshery.io).
 ## Join the service mesh community!
 
 <a name="contributing"></a><a name="community"></a>
-Our projects are community-built and welcome collaboration. 👍 Be sure to see the <a href="https://docs.google.com/document/d/17OPtDE_rdnPQxmk2Kauhm3GwXF1R5dZ3Cj8qZLKdo5E/edit">Layer5 Community Welcome Guide</a> for a tour of resources available to you and jump into our <a href="http://slack.layer5.io">Slack</a>!
+Our projects are a community-built and welcome collaboration. 👍 Be sure to see the <a href="https://docs.google.com/document/d/17OPtDE_rdnPQxmk2Kauhm3GwXF1R5dZ3Cj8qZLKdo5E/edit">Layer5 Community Welcome Guide</a> for a tour of resources available to you and jump into our <a href="http://slack.layer5.io">Slack</a>!
```
Comment (leecalcote) [inline README.md:38]: Thanks for this, @sonaljain067. The current grammar and meaning are correct as-is. Will you revert this particular change?

File: README.md [hunk 2]
```diff
@@ -69,11 +69,11 @@ Find out more on the <a href="https://layer5.io/community#meshmate">Layer5 commu
 
 ## Contributing to the Meshery.io Website
 
-Before contributing, please review the [Contribution Flow](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md). In the following steps you will set up your development environment, fork and clone the repository, run the site locally, and finally commit, sign-off, and push any changes made for review.
+Before contributing, please review the [Contribution Flow](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md). In the following steps, you will set up your development environment, fork and clone the repository, run the site locally, and finally commit, sign off, and push any changes made for review.
```
Comment (leecalcote) [inline README.md:72]:
```suggestion
Before contributing, please review the [Contribution Flow](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md). In the following steps, you will setup your development environment, fork and clone the repository, run the site locally, and finally commit, sign off, and push any changes made for review.
```

File: README.md [hunk 3]
```diff
@@ -69,11 +69,11 @@ Find out more on the <a href="https://layer5.io/community#meshmate">Layer5 commu
...
 
 ### 1. Set up your development environment
 
-- _The Meshery site is built using Jekyll - a simple static site generator! You can learn more about Jekyll and setting up your development environment in the [Jekyll Docs](https://jekyllrb.com/docs/)._
+- _The Meshery site is built using Jekyll - a simple static site generator! You can learn more about Jekyll and set up your development environment in the [Jekyll Docs](https://jekyllrb.com/docs/)._
```
Comment (leecalcote) [inline README.md:76]: Let's leave this as-is. You can learn about "setting up your development environment" in the Jekyll Docs, however, you can't "set up your development environment in the Jekyll Docs". You setup your development environment on your local machine.

File: README.md [hunk 4]
```diff
@@ -94,7 +94,7 @@ Before contributing, please review the [Contribution Flow](https://github.com/me
   ```bash
   $ make site
   ```
-  _Note: From the Makefile, this command is actually running `$ bundle exec jekyll serve --drafts --livereload`_
```
Comment (leecalcote) [inline README.md:97]: Given that this italicized note intends to convey a small insight to the reader, use of the word "actually" is befitting in that the assumption is that the reader didn't _actually_ realize what was happening behind the scenes. I suggest leaving as-is, unless you have separate justification...

**Comment by leecalcote** [comment]
@sonaljain067, most excellent to see you jump in here. Some feedback for you...

File: README.md [hunk 5]
```diff
@@ -110,4 +110,4 @@ Before contributing, please review the [Contribution Flow](https://github.com/me
 
 #### License
 
-This repository and site are available as open source under the terms of the [Apache 2.0 License](https://opensource.org/licenses/Apache-2.0).
\ No newline at end of file
+This repository and site are available as open-source under the terms of the [Apache 2.0 License](https://opensource.org/licenses/Apache-2.0).
```
Comment (leecalcote) [inline README.md:113]: No, ma'am. We do not hyphenate "open source". Please revert.

File: README.md [hunk 6]
```diff
@@ -26,7 +26,7 @@ _Visit [Meshery.io](https://meshery.io/) to learn more!_
 
 ## What is Meshery?
 
-Meshery is a multi-service mesh management plane adopting, operating, and developing on different service meshes. Meshery facilitates learning about functionality and performance of service meshes. It incorporates the collection and display of metrics from applications running on or across service meshes. At a high level, Meshery provides performance benchmarking, service mesh lifecycle management, and service mesh interoperability and federation.
+Meshery is a multi-service mesh management plane adopting, operating, and developing on different service meshes. Meshery facilitates learning about the functionality and performance of service meshes. It incorporates the collection and display of metrics from applications running on or across service meshes. At a high level, Meshery provides performance benchmarking, service mesh lifecycle management, and service mesh interoperability and federation.
```
Comment (leecalcote) [inline README.md:29]:
```suggestion
Meshery is the management plane for developing, deploying, and operating cloud native infrastructure and applications. Among other features, Meshery facilitates learning about the functionality and performance of service meshes. It incorporates the collection and display of metrics from applications running on or across service meshes. At a high level, Meshery provides performance benchmarking, service mesh lifecycle management, and service mesh interoperability and federation.
```

**Review by leecalcote — CHANGES_REQUESTED** [review]
Thanks much for this, @sonaljain067. 😄

File: README.md (hunk 1 above)
Comment (sonaljain067) [reply README.md:38]: Yeah

**Comment by sonaljain067** [comment]
Yeah, I would do all the changes.

**Comment by sonaljain067** [comment]
Done with the changes. Do I need to perform more of them?

**Comment by sonaljain067** [comment]
@leecalcote do I need to make more changes?

**Review by leecalcote — APPROVED** [review]
@sonaljain067, I think we're good to go. Thank you for this update. 👍

**Pull request merged**